test/FastTests/Corax/BoostingQuery.cs

```diff
@@ -214,7 +214,7 @@ public void OrderByBoostingTake4()
                 for (int i = 0; i < read; ++i)
                     sortedByCorax.Add(searcher.GetIdentityFor(ids[i]));
 
-                for (int i = 0; i < longList.Count; ++i)
+                for (int i = 0; i < 4; ++i)
```

Comment:
hmm?

Reply:
The name of the method is `Take4` :) the test was wrong.